Sources/TokamakStaticHTML/Modifiers/Effects/ScaleEffect.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 7/9/21.
+//
+
+import TokamakCore
+
+extension _ScaleEffect: DOMViewModifier {
+  public var attributes: [HTMLAttribute: String] {
+    ["style": "transform: scale(\(scale.width), \(scale.height));"]
```

Comment:
Does this mean that `anchor` property has no effect in the HTML renderer?

Reply:
Oh, yes. I'll add the `transform-origin` property.

Reply:
I fixed the rotationEffect modifier as well.